### ISSUE TITLE
Global - About us callouts - spacing

### ISSUE THREAD

**Issue by vcfolger** [vcfolger/southernicecreamtx.com#13]
This seems to be a global issue with the right side call-outs.

Add more space between image and red header. Also, font sizes in comps look a little different (header seems a touch bigger and body copy seems a touch smaller). Here is what I'm seeing.

![screen shot 2016-06-28 at 11 25 11 pm](https://cloud.githubusercontent.com/assets/20050245/16440481/e9f32992-3d87-11e6-9c26-0060634a2149.png)

**Comment by netintelpro** [comment]
"Add more space between image and red header. Also, font sizes in comps look a little different (header seems a touch bigger and body copy seems a touch smaller)"

These have been addressed.

I took pixel measurements of comps and applied percentages to actual elements. Though proportions of entire callouts may be tedious to apply. I can address this if its a deal breaker...but we may want to asses other priorities first.
